packages/blockchain-wallet-v4-frontend/src/data/middleware/webSocket/socketd/messageTypes.js

```diff
@@ -0,0 +1,19 @@
+export const header = message => message.entity === 'header' && message.header
+export const ethSentPending = message =>
+  message.transaction &&
+  message.address === message.transaction.from &&
+  message.transaction.state === 'pending'
+export const ethSentConfirmed = message =>
+  message.transaction &&
+  message.address === message.transaction.from &&
+  message.transaction.state === 'confirmed'
+export const ethReceivedPending = message =>
+  message.transaction &&
+  message.address === message.transaction.to &&
+  message.transaction.state === 'pending'
+export const ethReceivedConfirmed = message =>
+  message.transaction &&
+  message.address === message.transaction.to &&
+  message.transaction.state === 'confirmed'
+export const bitcoinTransaction = message =>
```

Comment:
prefer coin code (i.e BTC) over full name whenever we are using/referencing them in code